addons/point_of_sale/tests/test_point_of_sale_flow.py

```diff
@@ -1525,3 +1527,79 @@ def test_order_refund_with_invoice(self):
         credit_notes = self.env['account.move'].search([('move_type', '=', 'out_refund')], order='id desc', limit=1)
         self.assertEqual(credit_notes.ref, "Reversal of: "+invoices.name)
         self.assertEqual(credit_notes.reversed_entry_id.id, invoices.id)
+
```

Comment:
This diff is very different from the original PR. Maybe this test shouldn't be added here.